defaults/api_ents.go

```diff
@@ -0,0 +1,112 @@
+package defaults
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/moby/libentitlement/entitlement"
+	"github.com/moby/libentitlement/secprofile"
+)
+
+const (
+	// APIEntFullID is the API entitlement identifier; the value format is: "api.access:api-id:subset:[allow|deny]"
+	// ex: "api.access:engine.swarm:all:allow"
+	APIEntFullID = "api.access"
+
+	// APIFullControl specifies access control for the whole api
+	APIFullControl = "all"
+)
+
+// Default known APIs and API subsets to control access of
+const (
+	// EngineAPI defines the Moby-Engine API
+	EngineAPI = "engine"
```

Comment:
Good catch, let's create a set-able version variable so that Moby can tune it. Do you think we should accept different rules for different versions of the API?

Reply:
is there a valid scenario for running multiple versions of the API simultaneously?  I can't think of one, so I think it should just be a single version at a time.